lib/reporters/TapReporter.js

```diff
@@ -116,7 +119,46 @@ function prettyYamlValue (value, indent = 4) {
   }
 
   // Handle null, boolean, array, and object
-  return JSON.stringify(value, null, 2);
+  return JSON.stringify(decycledShallowClone(value), null, 2);
+}
+
+/**
+ * Creates a shallow clone of an object where cycles have
+ * been replaced with "[Circular]".
+ */
+function decycledShallowClone (object, ancestors = []) {
+  if (ancestors.includes(object)) {
```

Comment:
Use `Array#indexOf() !== -1` here for browser compat as I'd rather not ship a polyfill with this library, and given its low-level use in test runners it's not feasible for end-users to ship a polyfil either as they generally can't or wouldn't be expected to load something before this.

Reply:
Fixed. Should I also avoid `Array#map`?